pkg/platform/local/platform_test.go

```diff
@@ -166,6 +167,34 @@ func (suite *localPlatformTestSuite) TestResolveFunctionSpecRequestMemory() {
 	}
 }
 
+func (suite *localPlatformTestSuite) TestVeryBigFunction() {
+
+	// a small hack to set up required environment
+	_, _ = suite.platform.localStore.GetFunctions(&functionconfig.Meta{})
```

Comment:
what does the hack do here??

Reply:
@TomerShor spins up `nuclio-local-storage-reader` :magic_wand: